nginx_error_page/ny/error.html

```diff
@@ -0,0 +1,433 @@
+        <path stroke="rgba(0,0,0,0.8862745098039215)" d="M26 33h1" />
+        <path stroke="rgba(0,0,0,0.6039215686274509)" d="M27 33h1" />
+        <path stroke="rgba(0,0,0,0.10980392156862745)" d="M8 34h2m-2 1h1" />
+        <path stroke="#804970" d="M14 34h2m-2 1h2" />
+        <path stroke="#844b73" d="M16 34h2m-2 1h2" />
+        <path stroke="#6e3e60" d="M18 34h2m-2 1h2" />
+        <path stroke="rgba(0,0,0,0.8274509803921568)" d="M25 34h1" />
+        <path stroke="rgba(0,0,0,0.023529411764705882)" d="M26 34h1" />
+        <path stroke="rgba(0,0,0,0.5019607843137255)" d="M9 35h1m12 1h1" />
+        <path stroke="rgba(0,0,0,0.7098039215686275)" d="M24 35h1" />
+        <path stroke="rgba(0,0,0,0.5137254901960784)" d="M25 35h1m-5 2h1" />
+        <path stroke="rgba(0,0,0,0.9333333333333333)" d="M21 36h1" />
+        <path stroke="rgba(0,0,0,0.16470588235294117)" d="M23 36h1" />
+        <path stroke="rgba(0,0,0,0.23137254901960785)" d="M8 37h2" />
+        <path stroke="rgba(0,0,0,0.807843137254902)" d="M12 37h1" />
+        <path stroke="rgba(0,0,0,0.8549019607843137)" d="M13 37h1" />
+        <path stroke="rgba(0,0,0,0.9372549019607843)" d="M14 37h2" />
+        <path stroke="rgba(0,0,0,0.9411764705882353)" d="M16 37h2" />
+        <path stroke="rgba(0,0,0,0.8745098039215686)" d="M18 37h1" />
+      </svg>
+
+      <h1 class="header">
+        <div class="spinner" style="width: 18px; height: 18px;">
+          <svg class="spinner-svg" viewbox="25 25 50 50">
+            <circle class="spinner-circle-path" cx="50" cy="50" fill="none" r="20" stroke-miterlimit="10" stroke-width="4.5"></circle>
+          </svg>
+        </div>
+        Kutsekula tsamba la Kolibri
+      </h1>
+      <p class="message">Posachedwapa, musonyezedwa zoti muchite. Tangodikirani pang'ono</p>
+      <p><a href="https://community.learningequality.org/c/support/kolibri">Ngati ai, funsani anzanu m'magulu a pa intaneti</a></p>
+      <p><button class="button" onClick="location.reload();">Tsekuranisoni tsambali</button></p>
+      <p class="version">kolibri-server 1.2.3</p>
+    </div>
+
+    <script>
+      function retry() {
+        var req = new XMLHttpRequest();  // use it instead of fetch for IE11 compatibility
+        var latest_url = location.href;
+        req.open('GET', latest_url, true);
+        req.timeout = 1000;
+        req.onload = function() {
+          if (this.status >= 200 && this.status < 400) {
+            location.reload();
+          } else {
+            setTimeout(retry, 2000);
+          }
+        }
+        req.ontimeout = function() {
+          setTimeout(retry, 2000);
+        }
+        req.onerror = function() {
+          setTimeout(retry, 2000);
+        }
+        req.send(null);
+      }
+
+      retry();
+    </script>
+    <link href="https://fonts.googleapis.com/css?family=Noto+Sans:400,700&display=swap&subset=cyrillic,cyrillic-ext,devanagari,greek,greek-ext,latin-ext,vietnamese" rel="stylesheet">
```

Comment:
@indirectlylit @jredrejo noticing that the error pages have online content...

Reply:
@benjaoming  we know it. @indirectlylit added this code some time ago. I checked it and I can confirm you that:
a) these fonts are usually cached, most users won't do any external request and
b) I moved the line to the bottom of the file, so it does not block the page rendering. Users without Internet won't notice anything, the default fonts will play their role.